.github/workflows/pinot_tests.yml

```diff
@@ -72,16 +72,20 @@ jobs:
     if: github.repository == 'apache/pinot'
     runs-on: ubuntu-latest
     strategy:
+      # Changed to false in order to improve coverage using unsafe buffers
+      fail-fast: false
       matrix:
         testset: [ 1, 2 ]
-    name: Pinot Unit Test Set ${{ matrix.testset }}
+        java: [ 11, 17 ]
```

Comment:
do you want to add java 20 here as well?

Reply:
I've tried, but it seems that at least one maven plugin (spotless, IIRC) doesn't work in Java 20

Reply:
I've just tried `mvn verify -Dspotless.apply.skip -Dcheckstyle.skip -Dspotless.apply.skip -Dlicense.skip=true` with Java 20 and it seems that we have problems with Mockito.

We should upgrade to Mockito 5.x, which requires Java 11 and should work with modern Java versions. See https://github.com/mockito/mockito/releases/tag/v5.0.0